examples/configure-ingress-secrets-manager/output.tf

```diff
@@ -0,0 +1,39 @@
+# SM Instance
```

Comment:
generally looks good to me. only comment is I don't think you technically need these outputs (unless you're expecting this to be imported as a module into other terraform project).

main benefit would be exposing them via the CLI for the end user - but since some of them are marked sensitive, I don't know they provide much to the user.

if this is meant to be used as a module, then probably would make sense to move it to the `modules` directory, and then in `examples`, you could provide an example of using the module in the examples section.